packages/geo/src/lib/feature/shared/feature.utils.ts

```diff
@@ -109,10 +114,8 @@ export function featureFromOl(
 
   if (olLayer) {
     title = olLayer.get('title');
-    if (olLayer.get('sourceOptions')) {
-      exclude = olLayer.get('sourceOptions').excludeAttribute;
-      excludeOffline = olLayer.get('sourceOptions').excludeAttributeOffline;
-    }
+    exclude = olLayer.get('sourceOptions') ? olLayer.get('sourceOptions').excludeAttribute : undefined;
```

Comment:
Make a variable out of olLayer.get('sourceOptions')

Reply:
done